src/__tests__/questions.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Questions from '../Questions';
+
+describe('<Questions />', () => {
+  let output;
+  const listOfQuestions = [{
```

Comment:
What's the problem of using suitable method of Array.prototype to create mocks without hard coded duplicating of existing code?